doc/2/guides/getting-started/6-add-new-controller/index.md

```diff
@@ -4,4 +4,96 @@ type: page
 title: Add new Controller
 description: Extends Kuzzle API with new actions
 order: 600
----
\ No newline at end of file
+---
+
+# Add new Controller
+
+The Kuzzle API is composed of actions grouped in controllers.  
+A controller is a logical container that groups several actions together.
+
+Each action receives a Kuzzle request in parameter and is in charge of returning a result which will be transmitted in the response.
+
+## Register a new Controller
+
+We are going to modify the `app.ts` file to register a controller to expose new API actions.
+
+For this, we need to use the [Backend.controller.register](/core/2/some-link) method.
+
+This method takes the controller name and a [ControllerDefinition](/core/2/api/some-link) which defines the controller actions:
+
+
+```js
+app.controller.register('greeting', {
+  actions: {
+    sayHello: {
+      handler: async request => {
+        return `Hello, ${request.input.args.name}`;
+      }
+    }
+  }
+})
+```
+
+The code above will register a `greeting` controller with a `sayHello` action.  
+This action will use the `name` argument from the request and return a string.  
+
+::: info
+Kuzzle will generate a default HTTP route of the following format if none are provided:   
+`GET /_/<controller-name>/<action-name>`  
+Controller name and action name will be converted to `kebab-case`.
+:::
+
+We can now test our new action:
+
+:::: tabs
+::: tab Kourou
+
+```bash
+$ kourou greeting:sayHello --arg name=Yagmur
+[ℹ] Unknown command "greeting:sayHello", fallback to API method
+ 
+ 🚀 Kourou - Executes an API query.
+ 
+ [ℹ] Connecting to http://localhost:7512 ...
+ "Hello, Yagmur"
+ [✔] Successfully executed "greeting:sayHello"
+```
+
+:::
+::: tab HTTP
+
+```bash
+$ curl "http://localhost:7512/_/greeting/say-hello?name=Yagmur&pretty"
+{
+  "requestId": "383e016a-447e-4138-aa53-c07c9fd5c923",
+  "status": 200,
+  "error": null,
+  "controller": "greeting",
+  "action": "sayHello",
+  "collection": null,
+  "index": null,
+  "volatile": null,
+  "result": "Hello, undefined"
```

Comment:
So... I guess the controller didn't receive the `name` argument? :thinking:

Reply:
Ouuups :sweat_smile: